### PR TITLE
Add table and initial content to readme

### PR DESCRIPTION
**Pull request by izmalk**
## What is the goal of this PR?

To add initial content to the main README file in the drivers repository.

## What are the changes implemented in this PR?

Add some initial content and links to readme files, documentation, and source codes for language-specific drivers.

### CODE REVIEW

**Comment by izmalk** [comment]
Screenshot 2023-09-27 at 19 31 58

File: README.md [hunk 1]
```diff
@@ -5,3 +5,15 @@
 [![Discussion Forum](https://img.shields.io/discourse/https/forum.vaticle.com/topics.svg)](https://forum.vaticle.com)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typedb-796de3.svg)](https://stackoverflow.com/questions/tagged/typedb)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typeql-3dce8c.svg)](https://stackoverflow.com/questions/tagged/typeql)
+
+This repository stores all TypeDB Drivers built by Vaticle.
+
+See the table below for links to README files, documentation, and source codes.
```
Comment (benjaminasmall) [inline README.md:11]:
```suggestion
See the table below for links to README files, documentation, and source code.
```

File: README.md [hunk 2]
```diff
@@ -5,3 +5,15 @@
...
+| Python  | [README](https://github.com/vaticle/typedb-driver/tree/development/python/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/python/python-overview)   | `python/typedb` [directory](https://github.com/vaticle/typedb-driver/tree/development/python/typedb) |
+| Java    | [README](https://github.com/vaticle/typedb-driver/tree/development/java/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/java/java-overview)       | `java` [directory](https://github.com/vaticle/typedb-driver/tree/development/java)                   |
+| Node.js | [README](https://github.com/vaticle/typedb-driver/tree/development/nodejs/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/node-js/node-js-overview) | `nodejs` [directory](https://github.com/vaticle/typedb-driver/tree/development/nodejs)               |
+| C       |    |                                                                               | `c/src` [directory](https://github.com/vaticle/typedb-driver/tree/development/c/src)                 |
```
Comment (benjaminasmall) [inline README.md:19]: Please verify that this is the intended ordering of the five languages.

File: README.md [hunk 3]
```diff
@@ -5,3 +5,15 @@
...
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typeql-3dce8c.svg)](https://stackoverflow.com/questions/tagged/typeql)
+
+This repository stores all TypeDB Drivers built by Vaticle.
+
+See the table below for links to README files, documentation, and source codes.
+
+| Driver  | Readme                                                                             | Documentation                                                                 | Source codes                                                                                         |
+|---------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| Rust    | [README](https://github.com/vaticle/typedb-driver/tree/development/rust/README.md) |                                                                 | `rust/src` [directory](https://github.com/vaticle/typedb-driver/tree/development/rust/src)           |
```
Comment (benjaminasmall) [inline README.md:15]: Rust documentation will be out very soon.

**Review by benjaminasmall — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (izmalk) [reply README.md:15]: We can either 
- merge it now without the link and add the link later, when it's ready, or 
- add a link now, expecting it to be eventually working, but showing 404 for now. 
I'd prefer the first variant. What do you think?

File: README.md (hunk 2 above)
Comment (izmalk) [reply README.md:19]: Double-checked with Haikal, and he adjusted the order. Fixed in https://github.com/vaticle/typedb-driver/pull/453/commits/395640f51a726a661f11f2a2531c8fbb62bbaaea

File: README.md (hunk 3 above)
Comment (benjaminasmall) [reply README.md:15]: Definitely the former, please.

Comment (izmalk) [reply README.md]: I added a link to rust docs that do not exist yet - https://github.com/vaticle/typedb-driver/pull/453/commits/c3a6e092b602e3e83294e3287478356b384106f9

**Review by benjaminasmall — APPROVED** [review]

**Review by flyingsilverfin — APPROVED** [review]

**Pull request merged**